### PR TITLE
Update schema for monitoring_derived.airflow_task_instance_v1

### PR DESCRIPTION
**Pull request by lucia-vargas-a**
Addresses [Bug 1848852](https://bugzilla.mozilla.org/show_bug.cgi?id=1848852)
Change `pid` from INTEGER to STRING. Remove `updated_at` in schema currently saved as `job_id`.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-1408)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/query.sql [hunk 1]
```diff
@@ -7,7 +7,7 @@ SELECT
   start_date,
   end_date,
   duration,
-  updated_at job_id,
+  updated_at AS job_id,
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/query.sql:10]: updated_at as job_id? Seems like that's completely different pieces of information?

File: sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/schema.yaml [hunk 2]
```diff
@@ -19,10 +19,6 @@ fields:
   mode: NULLABLE
   name: try_number
   type: INTEGER
-- description:
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/schema.yaml:22]: Is this field not in the table?

**Review by kik-kik — CHANGES_REQUESTED** [review]

File: sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/schema.yaml (hunk 2 above)
Comment (lucia-vargas-a) [reply sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/schema.yaml:22]: It does, I just moved it below to match query and schema.

File: sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/query.sql (hunk 1 above)
Comment (lucia-vargas-a) [reply sql/moz-fx-data-shared-prod/monitoring_derived/airflow_task_instance_v1/query.sql:10]: Yeah, it's strange how it's renamed as `job_id` but it's not the idea to modify the logic in this PR, only to correct the update error, so I'm just using AS to highlight this renaming.

**Pull request merged**